conjur/api.py

```diff
@@ -285,6 +284,46 @@ def update_policy_file(self, policy_id, policy_file):
 
         return self._load_policy_file(policy_id, policy_file, HttpVerb.PATCH)
 
+    def rotate_anothers_api_key(self, resource, resource_to_rotate):
+        """
+        This method is used to rotate a user/host's API key
+        """
+        # Attach the resource type (user or host)
+        full_resource_id=':'.join([resource, resource_to_rotate])
+        query_params = {
+            'role': full_resource_id
+        }
+        response = invoke_endpoint(HttpVerb.PUT, ConjurEndpoint.ROTATE_API_KEY,
+                                   self._default_params,
+                                   api_token=self.api_token,
+                                   ssl_verify=self._ssl_verify,
+                                   query=query_params).text
+        return response
+
+    def rotate_personal_api_key(self, resource_to_rotate, current_password):
```

Comment:
isn't `resource_to_rotate` always equal `user`?
Why pass it if you know it?

Reply:
resource_to_rotate is not the actual resource but the name of the resource to rotate, for example (somehost, someuser). Fact that this is confused makes me want to change the name. I will change it to `id_of_logged_in_user`

Reply:
I think `user_name` is enough, but it's your call.
I just don't like using `id` as it is not clear.